docs/learn/submit-data.md

```diff
@@ -58,6 +58,50 @@ The `GasCostPerBlobByte` and `GasCostPerTransactionByte` are parameters that
 could potentially be adjusted through the system's governance mechanisms. Hence,
 actual costs may vary depending on the current state of these parameters.
 
+#### Gas fee calculation
+
+The total fee for a transaction is calculated as the product of the gas limit
+for the transaction and the gas price set by the user:
+
+$\text{Total Fee} = \text{Gas Limit} \times \text{Gas Price}$
+
+The gas limit for a transaction is the maximum amount of gas that a user is
+willing to spend on a transaction. It is determined by both a static
+"fixed cost" and a dynamic cost based on the size of each blob involved
+in the transaction:
+
+$\text{Gas Limit} = \text{Fixed Cost} + \sum_{i=1}^{n} \text{SparseSharesNeeded(Blob}_i) \times \text{Share Size} \times \text{Gas Cost Per Blob Byte}$
+
+Where:
+
+- $\text{Fixed Cost}$ is a static value (default is 65,000 gas)
```

Comment:
"Default" makes it sound like it can be changed somehow. Really this is a consensus parameter afaik.

```suggestion
- $\text{Fixed Cost}$ is a static value (65,000 gas)
```